databricks/koalas/tests/test_stats.py

```diff
@@ -80,7 +80,41 @@ def test_stat_functions_with_no_numeric_columns(self):
             }
         )
         kdf = ks.from_pandas(pdf)
-        self._test_stat_functions(pdf, kdf)
```

Comment:
Can you add a flag that that excludes `std` in the test instead of copying this method?

Reply:
But `std` is still valid for the `pandas < 1.2.0`.

Don't we need to care about that case ??

Reply:
Seems like this modification is not following the purpose of this test which is to check the behavior for non-numeric columns.

Maybe we should use string type data instead of date time here, and add a TODO to support `std` for the date time type. https://github.com/pandas-dev/pandas/pull/37436

Reply:
Addressed it. Thanks!